micro_epsilon_scancontrol_driver/include/micro_epsilon_scancontrol_driver/driver.h

```diff
@@ -30,14 +33,18 @@
 typedef pcl::PointCloud<pcl::PointXYZI> point_cloud_t;
 
 namespace scancontrol_driver
-{
-    class ScanControlDriver
+{   
+    // aliases
+    using set_duration_srv = micro_epsilon_scancontrol_msgs::srv::SetDuration;
```

Comment:
I was actually suggesting something like:
```
using SetDurationRequest = micro_epsilon_scancontrol_msgs::srv::SetDuration::Request
using GetDurationResponse = micro_epsilon_scancontrol_msgs::srv::GetDuration::Response
```
and I would place this in the class itself.

Reply:
Wouldn't this create the burden of creating a request response alias for every service?

Reply:
I guess... But I do it for readability and clarity.